core-services/retester/_config.yaml

```diff
@@ -13,6 +13,17 @@ retester:
       enabled: true
     openshift-knative:
       enabled: true
+    openshift-kni:
+      repos:
+        numaresources-operator:
+          enabled: true
+          max_retests_for_sha_and_base: 2
+        scheduler-plugins:
+          enabled: true
+          max_retests_for_sha_and_base: 2
+        cnf-features-deployer:
+          enabled: true
+          max_retests_for_sha_and_base: 2
```

Comment:
is this part of TAS?
once clarified LGTM

Reply:
no, forgot to save the new updates by mistake. should be cleaned up now